introduce new features and improve their documentation (#107)
georust/rinex: README.md

:construction: under development   
__CLI__ + :chart_with_upwards_trend: means record analysis is supported by the CLI, [README](rinex-cli/README.md)

> Files format

Reviewer: ```suggestion
File formats
```

  - the [DCB trait](https://doc.rs/rinex/latest/rinex/observation/Dcb.html)
  for Differential Code bias estimation
  - the [MP trait](https://doc.rs/rinex/latest/rinex/observation/Dcb.html)
  for Multi Path biases estimation
  - the [Iono Delay](https://doc.rs/rinex/latest/rinex/observation/IonoDelay.html) 
  estimator.

* `nav` : unlocks advanced NAV features, like
  - [Sv position](https://doc.rs/rinex/latest/rinex/navigation/Navigation.html) solver
  - [Elevation and azimuth](https://doc.rs/rinex/latest/rinex/navigation/Navigation.html) angles solver

* `processing` 
  - unlocks the [Preprocessing toolkit](https://doc.rs/rinex/latest/rinex/preprocessing.html) package,
  which contains several methods to resample, downsample, filter and sort RINEX datasets.

* `qc`: enables file Quality Check, ie., statistical analysis
on RINEX files. Reports are generated in HTML.
Complete QC is only feasible if you combine this feature to
the related type feature, `obs` + `nav` is the typical combination.

<img align="right" width="400" src="https://upload.wikimedia.org/wikipedia/commons/4/46/SBAS_Service_Areas.png">

> * `sbas`: SBAS (stationnary augmentation systems) selection

Reviewer: ```suggestion
* `sbas`: SBAS (stationary augmentation systems) selection
```

processing/esbc00dnkr2021/mask:sv | 341.42 ms | 
processing/esbc00dnkr2021/smooth:hatch:l1c,l2c | 502.90 ms | 

Contributions
=============

Contributions are welcomed, do not hesitate to open new issues
and submit Pull Requests through Github.

> If you want to take part in active developments, checkout our [contribution guidelines and hints](CONTRIBUTE.md) to navigate this library quicker.

Reviewer: ```suggestion
If you want to take part in active developments, check out our [contribution guidelines and hints](CONTRIBUTE.md) to navigate this library quicker.
```
